app/gateway-oss/2.4.x/hybrid-mode.md

```diff
@@ -290,6 +290,27 @@ cluster_cert_key = data-plane.crt
 `cluster_server_name` specifies the SNI (Server Name Indication extension) to use for data plane
 connections to the control plane through TLS. When not set, data plane will use `kong_clustering` as the SNI.
 
+## Revocation check of Data Plane certificates
+
+When Kong is running Hybrid mode with PKI support, the Control Plane can be configured to
```

Comment:
we use the wording `PKI mode` above, should use the same description for matching (but having two "mode" in one sentence then feels verbose to me..)
```suggestion
When Kong is running Hybrid mode with PKI mode, the Control Plane can be configured to
```